include/TPP/Passes.td

```diff
@@ -155,7 +155,8 @@ def TileConsumerAndFuseProducers : Pass<"tile-consumer-and-fuse-producers",
            "Run fusion for the given number of iterations">,
     Option<"useForAll", "use-for-all", "bool", "true", "Use parallel forAll">,
     Option<"minTileFactor", "min-tile-factor", "int64_t", "2",
-           "Minimum factor between dimension size and a tile size">
+           "Minimum factor between dimension size and a tile size">,
+    Option<"keepNamedOp", "keep-named-op", "bool", "false", "Don't generalize named ops">
```

Comment:
Looks like the option is no longer really needed for anything. Could you remove it from this PR?
I would recommend opening a separate PR that removes `populateLinalgNamedOpsGeneralizationPatterns` all together from tile and fuse.